bin/BuildPackages.sh

```diff
@@ -279,8 +248,8 @@ build_one_package() {
   ) || build_fail
 }
 
-date >> "$LOGDIR/$FAILPKGFILE.log"
-for PKG in "${PACKAGES[@]}"
+date >> "$LOGDIR/fail.log"
+for PKG in ${PACKAGES}
```

Comment:
Plus, package build systems are quite likely to break due to spaces in their absolute paths, too...
And I think no regular user will try to rename their package directories, let alone insert spaces... so this feels a lot like hypothetical bike shedding to me... Anyway, as I said, I'll restore the array code, then if anybody feels this is something important, they can add whitespace-paths upport in another PR (but please actually test it)